static/templates/stream_settings/stream_subscription_request_result.hbs

```diff
@@ -3,10 +3,10 @@
 <br />
 {{/if}}
 {{#if subscribed_users}}
+    {{#if subscribed_users.[1]}} {{t "Successfully subscribed users:" }}
+    {{else}} {{t "Successfully subscribed user:" }}
```

Comment:
Let's put the strings on their own lines, with property indentation; it'll be easier to read.

Reply:
> Let's put the strings on their own lines, with property indentation; it'll be easier to read.

I've updated the code. Could you please check if it matches the requirement?